script/sheet/aptitude.js

```diff
@@ -0,0 +1,35 @@
+import { DarkHeresyItemSheet } from "./item.js";
+
+export class AptitudeSheet extends DarkHeresyItemSheet {
+    static get defaultOptions() {
+        return mergeObject(super.defaultOptions, {
+            classes: ["dark-heresy", "sheet", "aptitude"],
+            template: "systems/dark-heresy/template/sheet/aptitude.html",
+            width: 500,
+            height: 369,
+            resizable: false,
+            tabs: [
+                {
+                    navSelector: ".sheet-tabs",
+                    contentSelector: ".sheet-body",
+                    initial: "stats",
+                },
+            ]
+        });
+    }
+
+    getData() {
+        const data = super.getData();
+        return data;
+    }
+
+    _getHeaderButtons() {
+        let buttons = super._getHeaderButtons();
+        buttons = [].concat(buttons);
+        return buttons;
+    }
+
+    activateListeners(html) {
+        super.activateListeners(html);
+    }
```

Comment:
Really minor thing, but I think these functions could be removed without breaking anything.

Reply:
Thanks for having a look!

I see the same code in all .js classes that extend DarkHeresyItemSheet - so these methods can be removed form all of them?

Reply:
Correct.